site/content/docs/user/known-issues.md

```diff
@@ -17,12 +17,39 @@ It may additionally be helpful to:
 - reach out and ask for help in [#kind] on the [kubernetes slack]
 
 ## Contents
+* [Failures involving mismatched kubectl versions](#failures-involving-mismatched-kubectl-version)
 * [Docker on Btrfs](#docker-on-btrfs)
 * [Docker installed with Snap](#docker-installed-with-snap)
 * [Failing to apply overlay network](#failing-to-apply-overlay-network)
 * [Failure to build node image](#failure-to-build-node-image)
 * [Failure for cluster to properly start](#failure-for-cluster-to-properly-start)
 
+## Failures involving mistmatched kubectl versions
+
```

Comment:
I think that we could use an introductory line explaining the context , something along the lines `If you have the following errors using kind:`

Reply:
That sounds like a great idea ✨ will update

Reply:
or would ``If you see something like the following error message:`` be better, as I'm not confident the error looks the absolute same every time, just some similarities

Reply:
yeah, much better than mine